scripts/pre_push_hook.py

```diff
@@ -295,21 +295,21 @@ def _install_hook():
         print 'Copied file to .git/hooks directory'
 
 
-def does_diff_include_js_files(files_to_lint):
-    """Returns true if diff includes JS files.
+def does_diff_include_ts_files(files_to_lint):
+    """Returns true if diff includes TS files.
 
     Args:
         files_to_lint: list(str). List of files to be linted.
 
     Returns:
-        bool. Status of JS files in diff.
+        bool. Status of TS files in diff.
     """
 
-    js_files_to_check = [
+    ts_files_to_check = [
         filename for filename in files_to_lint if
-        filename.endswith('.js')]
+        filename.endswith('.ts')]
```

Comment:
@ankita240796 is there any .js file for which we need to run the frontend test? 
What I see:
- .js file for e2e tests
- contant.js
- rich_text_components_definitions.js

Reply:
There is `ExpressionParserService.js` as well.

Reply:
- .js file for e2e tests: We expect that changes in these files should not trigger frontend tests.
- contant.js: We haven't used this value in the frontend as this value is sent via the server as a json, right? If yes then we are not expecting to trigger frontend test for any changes in this file.
- rich_text_components_definitions.js: This is also used as a json.

-  ExpressionParserService.js: We don't expect any changes in this file and this is also been ignored for eslint.

@ankita240796, So going through all this I'm not sure whether we should trigger frontend tests for .js file change? Let me know if I have missed something?

Reply:
There can be cases when the ParserService is updated if we change the grammar spec. I am not sure how frequent that can be. If that is not at all expected, it is fine to have ts filenames only.

Reply:
```
How to update the parser:
1. Modify parser.pegjs.
2. Add tests in test.js for the new or modified grammar.
3. Run

      bash scripts/create_expression_parser.sh

   which updates ExpressionParserService.js. Then run the frontend unit tests using

      bash scripts/run_frontend_tests.sh

   to ensure that the new grammar passes the tests in ExpressionParserServiceSpec.js and
   ExpressionEvaluatorServiceSpec.js.
4. Repeat 1-3 until the desired grammar is written and tested.
5. Check the changes. You need to check in parser.pegjs as well as ExpressionParserService.js
  (and probably ExpressionParserServiceSpec.js and ExpressionEvaluatorServiceSpec.js).
```

@ankita240796 I found the above documentation [here](https://github.com/oppia/oppia/tree/develop/core/templates/dev/head/expressions/README.txt).
So going through this I think we don't expect developers to change the parsers directly also, this file is ignored by the eslint.

Reply:
Ah ok, then I think it is fine to have only ts filenames!